### DISCUSSION TITLE
how does the remove miner button work?

### DISCUSSION THREAD

**Discussion by bomba2006** [UnamSanctam/UnamWebPanel#326] [Q&A] [answered]
does the remove miner button in the Unam Web Panel also uninstalls the miner in the computer if the startup option is enabled?

**Comment by UnamSanctam** [comment, marked as answer]
No it only removes it from the list, the miners don't have any uninstallation code inside of them. For many reasons reasons, one is sp that no one can uninstall your miner if they were to gain access to your web panel.
